client/src/modules/templates/bhEmployeeSelect.tmpl.html

```diff
@@ -0,0 +1,24 @@
+<div ng-form="{{ $ctrl.name }}" bh-employee-select ng-model-options="{ updateOn: 'default' }">
+  <div
+    class="form-group"
+    ng-class="{ 'has-error' : EmployeeForm.$$parent.$submitted && EmployeeForm.employee_id.$invalid }">
```

Comment:
`$$parent` doesn't exist on the form - it should be `$$parentForm`.  [See this answer for more info](https://stackoverflow.com/a/31984524/6251968)